.github/workflows/build-base-image.yml

```diff
@@ -15,15 +15,16 @@ jobs:
         uses: docker/setup-qemu-action@v2
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
-      - name: Login to Docker Hub
+      - name: Login to ghcr.io
         uses: docker/login-action@v2
         with:
-          username: signalkci
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          registry: ghcr.io
+          username: ${{ secrets.GHCR_USERNAME }}
+          password: ${{ secrets.GHCR_TOKEN }}
```

Comment:
We already have RELEASE_PAT. PATs have finite lifetime, I think it would make sense to use just a single PAT to ease keeping them valid. Call it `GITHUB_PAT` and document the needed access right here in a comment?

Reply:
Is this OK?
`username: ${{ github.actor }}`
`password: ${{ secrets.GITHUB_PAT }} # requires a write access to write:packages`